### PR TITLE
Setup MongoDB for Customer service

### LINKED ISSUE #9: Set up Mongo DB

### PR DESCRIPTION
**Pull request by JustMilan**
## Changes
A short description about the overall changes you've made to the project.

## Checklist
- [X] Implement MongoDB
- [X] Add/edit tests
- [X] Format all modified files
- [X] Check for missing newlines
- [X] Add repositories
- [X] Add Document annotations

## When merging, this pull request 
closes #9

### CODE REVIEW

File: customer/src/test/java/nl/softwarestrijders/waiter/customer/core/domain/CustomerTest.java [hunk 1]
```diff
@@ -76,7 +78,7 @@ void shouldThrowInvalidEmailExceptionIfEmailHasDoubleDot() {
 	@DisplayName("Getters and setters test for JaCoCo")
 	void getterSetterTests() {
 		var customer = new Customer("Test", "Test", "test@test.nl", address);
-		assertDoesNotThrow(() -> customer.setId("1"));
+		assertDoesNotThrow(() -> customer.setId(new UUID(1, 1)));
```
Comment (xandervedder) [inline customer/src/test/java/nl/softwarestrijders/waiter/customer/core/domain/CustomerTest.java:81]: might wanna make this UUID in the constructor of the customer, btw `UUID.createRandomUUID()` (static method) is preferred

**Review by xandervedder — CHANGES_REQUESTED** [review]

File: customer/src/test/java/nl/softwarestrijders/waiter/customer/core/domain/CustomerTest.java (hunk 1 above)
Comment (JustMilan) [reply customer/src/test/java/nl/softwarestrijders/waiter/customer/core/domain/CustomerTest.java:81]: Good one, was questioning myself to do so

**Review by Rubenvdbrink — APPROVED** [review]
👀✅2me

**Comment by Rubenvdbrink** [comment]
image

**Review by xandervedder — APPROVED** [review]
You should probably think about using `UUID.randomUUID()` in the constructor of both of your domain classes, but that can be done later.

**Pull request merged**